blocks/form/form.js

```diff
@@ -0,0 +1,135 @@
+function constructPayload(form) {
+  const payload = {};
+  [...form.elements].forEach((fe) => {
```

Comment:
this looks like a good candidate for `reduce`

Reply:
Done.